docs/dashboard/configuration/channels.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Channels
+---
+
+This page allows you to define and manage your sales channels for your store. You can create multiple channels with different currencies.
```

Comment:
Missing information about what is a channel.

Reply:
I'll create an additional section called "Channels" in the main menu of the dashboard docs where it will be explained.